### ISSUE TITLE
log which container is being operated on in trace

### ISSUE THREAD

**Issue by hickeng** [vmware/vic#1865]
Currently the trace output doesn't include which container is being operated on. This makes it almost impossible to trace concurrent operations, and awkward to map even serial operations across components:

Portlayer:

```
time="2016-08-05T23:12:16Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*ContainersHandlersImpl).CommitHandler] Containers.CommitHandler" 
time="2016-08-05T23:12:16Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/portlayer/exec.(*Container).Commit] Committing handle" 
time="2016-08-05T23:12:16Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/portlayer/exec.(*Container).stop] Container.stop" 
time="2016-08-05T23:12:16Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/exec.(*Container).stop] [916.358002ms] Container.stop" 
time="2016-08-05T23:12:17Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/portlayer/exec.(*Container).Commit] [1.275672261s] Committing handle" 
time="2016-08-05T23:12:17Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/portlayer/restapi/handlers.(*ContainersHandlersImpl).CommitHandler] [1.276749265s] Containers.CommitHandler" 
```

Personality:

```
time="2016-08-05T23:09:37Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/engine/backends.(*Container).containerCreate] [509.585876ms] Container.containerCreate" 
time="2016-08-05T23:09:37Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/engine/backends.(*Container).ContainerCreate] [510.356025ms] ContainerCreate" 
time="2016-08-05T23:09:37Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/apiservers/engine/backends.(*Container).ContainerStart] ContainerStart" 
time="2016-08-05T23:09:47Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/engine/backends.(*Container).ContainerStart] [10.576098821s] ContainerStart" 
time="2016-08-05T23:09:47Z" level=debug msg="[BEGIN] [github.com/vmware/vic/lib/apiservers/engine/backends.(*Container).ContainerStop] ContainerStop" 
time="2016-08-05T23:10:34Z" level=debug msg="[ END ] [github.com/vmware/vic/lib/apiservers/engine/backends.(*Container).ContainerStop] [46.635692084s] ContainerStop" 
```

**Comment by matthewavery** [comment]
After looking through old closed tickets I have found #2494 which I suspect has actually already met the spirit of this ticket. As such I suspect that #3170 may be over kill and upon input from @hickeng may be closed with no merge.

**Comment by mdubya66** [comment]
pulling as too risky. will re-think for 0.9
